uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document.spec.js

```diff
@@ -318,6 +318,122 @@ define([
           expect(DocumentService.save).toHaveBeenCalledWith(jasmine.objectContaining({ activity_date_time: '' }));
         });
       });
+
+      describe('document contains files', function () {
+        beforeEach(function () {
+          angular.extend(controller.document, mockDocument);
+        });
+
```

Comment:
Inconsistent spacing between `beforeEach` and `describe` blocks. Please fix it in all places in the document.